### DISCUSSION TITLE
Universal Control Detection

### DISCUSSION THREAD

**Discussion by ryanseddon** [Hammerspoon/hammerspoon#3339] [General]
I've started using the great [universal control feature](https://support.apple.com/en-au/HT212757) on my macs/iPad. I've got a really great setup where my keyboard, mouse and audio all works nicely no matter which mac I'm controlling. 

The audio is manually configured by making my main mac an airplay receiver and then piping my universally controlled mac to airplay to that.

I'm wondering if you can detect if a mac is being controlled by universal control? And vice versa where I can know if the cursor has gone off screen to another device.

The automation I'd want to do is:
- When host machine hands over control to the other device turn audio up 100% 
- On secondary device make sure it's airplaying audio to host mac
- Then on coming back to host machine restore audio to previous setting

**Comment by luigibertaco** [comment]
This would be very nice, also the possibility to switch focus to the secondary device with a keyboard shortcut would be pretty handy.
